Dockerfile.app

```diff
@@ -71,5 +71,8 @@ FROM python-base as production
 # Copy artifacts from other stages
 COPY --from=venv-builder /usr/src/app /usr/src/app
 
+ENV ADALA_LOG_LEVEL=DEBUG
+ENV LITELLM_LOG=WARNING
+
```

Comment:
should this be in the docker-compose instead of the Dockerfile? I wouldn't necessarily think to look for it here. And also need to make sure the appropriate env var is set for dev+prod in the infra repo if necessary (debug for dev, info for prod)

Reply:
Ah, yeah, was going to set to INFO before releasing, forgot to put a comment about this.
I guess I'd usually look here before docker-compose but don't mind much either way if you think docker-compose is more obvious.

Reply:
docker-compose makes more sense to me because you can change the setting without rebuilding, and also we sometimes end up editing the compose files anyway locally due to Mac chicanery